cypress/e2e/11-unarchive_patient.cy.js

```diff
@@ -0,0 +1,114 @@
+const {graphqlURL} = require("../utils/request-utils");
```

Comment:
⚠️ once the Boban's jest-axe changes merge in, should I remove this cypress test and add testing in the frontend tests or leave this as-is? thoughts? 🤔

Reply:
Since this isn't a main flow I'd think that jest-axe would be sufficient.

Reply:
ok! let me update this!! :)

Reply:
i realized... once yours is merged in haha 😅